packages/taco/integration-test/condition-lingo.test.ts

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios';
+import * as https from 'https';
+import { describe, test } from 'vitest';
+import { CompoundConditionType } from '../src/conditions/compound-condition';
+import { ConditionExpression } from '../src/conditions/condition-expr';
+import { IfThenElseConditionType } from '../src/conditions/if-then-else-condition';
+import { SequentialCondition } from '../src/conditions/sequential';
+import {
+  testContractConditionObj,
+  testJsonApiConditionObj,
+  testJsonRpcConditionObj,
+  testJWTConditionObj,
+  testRpcConditionObj,
+  testSigningObjectAbiAttributeConditionObj,
+  testSigningObjectAttributeConditionObj,
+  testTimeConditionObj,
+} from '../test/test-utils';
+
+const LYNX_NODES = [
+  'https://lynx-1.nucypher.network',
+  'https://lynx-2.nucypher.network',
+  'https://lynx-3.nucypher.network',
+];
+
+async function validateConditionExpression(
+  conditionExpr: ConditionExpression,
+): Promise<void> {
+  const lynxNode = LYNX_NODES[Math.floor(Math.random() * LYNX_NODES.length)];
+  const httpsAgent = new https.Agent({
+    rejectUnauthorized: false,
+  });
+
+  const response = await axios.post(
+    `${lynxNode}:9151/validate_condition_lingo`,
+    JSON.stringify(conditionExpr.toJson()),
+    {
+      httpsAgent,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    },
+  );
+  if (response.status !== 200) {
+    throw new Error(
+      `Request failed with status ${response.status}: ${JSON.stringify(response.data)}`,
+    );
+  }
+  return;
+}
+
+describe.skipIf(!process.env.RUNNING_IN_CI)(
```

Comment:
Out of curiosity: why not run this test in CI? Is this only intended to be executed locally?

Reply:
ooh, right. I understood the opposite indeed

Reply:
It's kind of a double negative expression ("_skip if not running in CI_"), maybe a comment can help to clarify